tests/unit_tests/main_test.cpp

```diff
@@ -112,11 +112,15 @@ int main(int argc, char** argv) {
                         if (dev.is_gpu() && vendor_id == INTEL_ID)
                             continue;
 #endif
-#if !defined(ENABLE_CUBLAS_BACKEND) && !defined(ENABLE_CUSOLVER_BACKEND) && \
-    !defined(ENABLE_CURAND_BACKEND) && !defined(ENABLE_ROCBLAS_BACKEND)
+#if !defined(ENABLE_CUBLAS_BACKEND) && !defined(ENABLE_CURAND_BACKEND) && \
+    !defined(ENABLE_CUSOLVER_BACKEND)&&
```

Comment:
This does not compile. Can you please re-built, test and upload new log files?

Reply:
Sorry. We had some server problems yesterday, fixed and run tests
[rocrng_ct_tests.txt](https://github.com/oneapi-src/oneMKL/files/8664046/rocrng_ct_tests.txt)
.